frontend/src/components/TopBar.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components/macro';
+
+
+const TopBar = ()=>{
+
+    return(
+        <TopBarContainer>
+            Hello
+        </TopBarContainer>
+    )
+}
+
+const TopBarContainer = styled.div`
+  width: 100%;
+  background-color: #61dafb;
+  height: 3em;
+`
+
+export default TopBar
```

Comment:
if you only need styling you can also export your `styled.div` directly and use it like a regular tag elsewhere e.g. to include text 
`<TopBar>Hello</TopBar>`

```suggestion
import styled from 'styled-components/macro';

export default styled.div`
  width: 100%;
  background-color: #61dafb;
  height: 3em;
`
```